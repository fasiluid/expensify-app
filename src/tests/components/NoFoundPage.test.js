import React from "react";
import { shallow } from "enzyme";
import NoFoundPage from "../../components/NoFoundPage";

test('should render NoFoundPage correctly', () => {
    const wrapper = shallow(<NoFoundPage />);
    expect(wrapper).toMatchSnapshot();
});