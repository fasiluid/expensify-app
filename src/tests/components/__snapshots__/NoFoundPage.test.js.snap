// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoFoundPage correctly 1`] = `
<div>
  404! 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
